java-manta-client/pom.xml

```diff
@@ -170,6 +170,20 @@
             <version>${dependency.urlbuilder.version}</version>
         </dependency>
 
+        <!-- object cloning library for memoization of encryption state -->
```

Comment:
Can we include a "golden file" to deserialize as part of the kyro tests?  That way  we can be sure that version X objects can still be read if we bump the library dep to X+1.

Regarding shading/relocating.  I think the stance that java-manta has taken so far is to do so wherever possible, so I would start with that as the default position.